README.md

```diff
@@ -149,9 +149,9 @@ To instrument tracers and meters using the [opentelemetry-javaagent](https://git
 
 ```bash
 java -javaagent:path/to/opentelemetry-javaagent-<version>-all.jar \
-     -Dota.exporter.jar=path/to/opentelemetry-exporter-newrelic-auto-<version>.jar \
-     -Dota.exporter.newrelic.api.key=${INSIGHTS_INSERT_KEY} \
-     -Dota.exporter.newrelic.service.name=best-service-ever \
+     -Dotel.exporter.jar=path/to/opentelemetry-exporter-newrelic-auto-<version>.jar \
+     -Dotel.exporter.newrelic.api.key=${INSIGHTS_INSERT_KEY} \
+     -Dotel.exporter.newrelic.service.name=best-service-ever \
```

Comment:
this name change was the main thing relevant here.

Reply:
That's better than "over the air" as before.